.gitignore

```diff
@@ -52,6 +53,7 @@ __pycache__
 **/node_modules/**
 /tmp
 **/npm-debug.log*
+**/package-lock.json
```

Comment:
Should we remove package-lock here?

Reply:
I added this to our `.gitignore` because we've had dependency upgrade issues in the past. We already have a `package-lock.json` committed to our source files and should only be committing new `package-lock.json` files if a dependency is intentionally changed/upgraded as part of the feature work.